cargo-nextest/src/dispatch.rs

```diff
@@ -83,6 +84,27 @@ impl ConfigOpts {
 
 #[derive(Debug, Subcommand)]
 enum Command {
+    /// List test binaries in workspace
+    ///
+    /// This command builds test binaries and list them.
+    /// Use --message-format json to get machine-readable output.
+    ///
+    /// For more information, see <https://nexte.st/book/listing>.
+    ListBins {
```

Comment:
Thinking about this more, I think maybe this should be part of `list`, as a `--no-query` (for "do not run/query binaries") option.

Also, `list` should accept a `--bin-list` parameter as well, just like the `run` command does.

Reply:
`list --bin-list <path>` ? I was going toward this at first (then I read you comment on the ticket again), so OK for me.

Reply:
Yeah I think that makes sense because you may want to list out all the tests given a list of binaries.

However it occurs to me that maybe we shouldn't call the parameter --bin-list then to avoid confusion.  Maybe something like --binary-metadata? Happy to hear your suggestions too.

Reply:
Did a bit differently (see new comment).